Include/pymacro.h

```diff
@@ -102,4 +102,15 @@
 
 #define Py_UNREACHABLE() abort()
 
+
+/* If we're using GCC, use __builtin_expect() */
+#if defined(__GNUC__) && (__GNUC__ > 2)
```

Comment:
I think `__GNUC__ > 2` is redundant.

Reply:
Redundant because Python won't compile with GCC 2.x anyway?

Reply:
Because:

* I don't know GCC 2.x supports all of C99 features we use in PEP 7.
* I don't know how to use GCC 2 to compile Python in recent days.
* I don't know who try to compile Python with GCC 2.

Reply:
I found this page: https://gcc.gnu.org/c99status.html
We use some GCC>=3.0 features already.

* designated initializers
* new block scopes for selection and iteration statements
* stdbool.h (GCC 2.95 had `<stdbool.h>`, but based on an early draft with major differences from C99 semantics.)